src/Quiz.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import IntervalExercise  from './IntervalExercise.js'
+import PropTypes from "prop-types";
+
+function ContinueButton({ onClick }) {
+    return (
+        <button onClick={onClick}> Continue </button>
+    )
+}
+
+class Quiz extends Component {
+    render() {
+        var maybeContinueButton;
+        if (// we've answered correctly
+            this.props.submittedAnswers.includes(this.props.interval.toString())
+            // and this is not the last exercise
+            && this.props.currentExercise < this.props.numExercises - 1
+          ) {
```

Comment:
Indentation note: I would do this like
```suggestion
        if (
            // we've answered correctly
            this.props.submittedAnswers.includes(this.props.interval.toString())
            // and this is not the last exercise
            && this.props.currentExercise < this.props.numExercises - 1
        ) {
```

instead